DataConnectors/CEF/cef_AMA_troubleshoot.py

```diff
@@ -621,39 +640,43 @@ def incoming_logs_validations(self, mock_message=False):
         poll_obj = select.poll()
         poll_obj.register(tcp_dump.stdout, select.POLLIN)
         while (end_seconds - start_seconds) < tcpdump_time_restriction:
-            if mock_message is True:
+            if mock_message is True and mock_message_counter < mock_message_max:
                 # Sending mock messages
                 mock_message_counter += 1
-                self.send_cef_message_local(514, 1)
-            poll_result = poll_obj.poll(0)
+                self.send_message_local(514, 2)
+            poll_result = poll_obj.poll(2500)
             if poll_result:
-                line = str(tcp_dump.stdout.readline().decode('utf-8').strip("\n"))
-                if self.handle_tcpdump_line(line):
-                    command_object.command_result = line
-                    command_object.run_full_verification()
-                    print_ok(self.CEF_EVENTS_FOUND_MESSAGE)
-                    return True
+                while line:
+                    line = tcp_dump.stdout.readline().decode('utf-8').strip("\n")
+                    if not line:
+                        break
+                    if self.handle_tcpdump_line(line, self.IDENT_NAME[STREAM_SCENARIO]):
+                        command_object.command_result = line
+                        command_object.run_full_verification()
+                        print_ok("Found {0} in stream. Please verify {0} events arrived at your workspace".format(
+                        STREAM_SCENARIO.upper()))
+                        return True
             end_seconds = int(round(time.time()))
-        print_error(self.CEF_EVENTS_NOT_FOUND_ERROR_MESSAGE)
+        print_error("Could not locate {0} message in tcpdump. Please verify {0} events can be sent to the machine and"
+                    " there is not firewall blocking incoming traffic".format(STREAM_SCENARIO.upper()))
         command_object.command_result = str(line)
         command_object.run_full_verification()
         return False
 
-    def send_cef_message_local(self, port, amount):
+    def send_message_local(self, port, amount):
         """
         Generate local CEF events in a given amount to a given port
         :param port: A destination port to send the events
         :param amount: The amount of events to send
         """
         try:
             for index in range(0, amount):
-                command_tokens = ["logger", "-p", "local4.warn", "-t", "CEF:", self.FIXED_CEF_MESSAGE, "-P", str(port),
-                                  "-n",
-                                  "127.0.0.1"]
+                command_tokens = ["logger", "-p", "local4.warn", "-t", self.IDENT_NAME[STREAM_SCENARIO],
+                self.STREAM_MESSAGE[STREAM_SCENARIO], "--rfc3164", "-P", str(port), "-n", "127.0.0.1"]
```

Comment:
--rfc3164 is new, why is it needed?

Reply:
To uphold to the AMA RFC enforcing standard and match our documentation.